package.json

```diff
@@ -7,14 +7,18 @@
     "query-string": "^6.9.0"
   },
   "scripts": {
-    "build:webpack": "calypso-build --config='./webpack.config.js'",
-    "clean": "rm -rf assets/dist/ assets/release/",
+    "build:js": "calypso-build",
+    "build": "run-p \"build:*\"",
+    "clean": "rm -rf assets/dist/",
+    "dev:js": "calypso-build --watch",
+    "dev": "run-p \"dev:*\"",
+    "start": "npm run dev",
     "release:build-all": "composer install --no-dev && NODE_ENV=production run-p \"build:webpack\"",
```

Comment:
We should also update the `"release:build-all"` script, and replace the use of `build:webpack` (which has been removed) with `build`, or `build:js` (in which case we could get rid of `build`). I'd suggest to replace it with this:
```
    "release:build-all": "composer install --no-dev && NODE_ENV=production run-p \"build\"",
```

Reply:
Good catch! Fixed in https://github.com/Automattic/newspack-content-converter/pull/39/commits/0ece99160230220eb81092888cdbe0453c162c4e